.licenserc.yaml

```
@@ -5,13 +5,13 @@ header:
       
       This source code is licensed under the MIT license found in the
       LICENSE file in the root directory of this source tree.
-  alternatives:
-    - content: |
-        Copyright (c) Meta Platforms, Inc. and affiliates.
-        All rights reserved.
-        
-        This source code is licensed under the license found in the
-        LICENSE file in the root directory of this source tree.
+  alternative:
```

```suggestion
    alternatives:
       - |
        Copyright (c) Meta Platforms, Inc. and affiliates.
        All rights reserved.
        
        This source code is licensed under the license found in the
        LICENSE file in the root directory of this source tree.
```

Not sure about indentation, can't see well here. But "alternatives" should be aligned with "content"